azure-keyvault-webkey/pom.xml

```diff
@@ -1,117 +1,127 @@
+		<version>1.1-beta-1</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
 
-  <artifactId>azure-keyvault-webkey</artifactId>
-  <packaging>jar</packaging>
+	<artifactId>azure-keyvault-webkey</artifactId>
+	<packaging>jar</packaging>
 
-  <name>Microsoft Azure SDK for Key Vault WebKey</name>
-  <description>This package contains Microsoft Azure Key Vault WebKey library.</description>
-  <url>https://github.com/Azure/azure-sdk-for-java</url>
+	<name>Microsoft Azure SDK for Key Vault WebKey</name>
+	<description>This package contains Microsoft Azure Key Vault WebKey library.</description>
+	<url>https://github.com/Azure/azure-sdk-for-java</url>
 
-  <licenses>
-    <license>
-      <name>The MIT License (MIT)</name>
-      <url>http://opensource.org/licenses/MIT</url>
-      <distribution>repo</distribution>
-    </license>
-  </licenses>
+	<licenses>
+		<license>
+			<name>The MIT License (MIT)</name>
+			<url>http://opensource.org/licenses/MIT</url>
+			<distribution>repo</distribution>
+		</license>
+	</licenses>
 
-  <scm>
-    <url>scm:git:https://github.com/Azure/azure-sdk-for-java</url>
-    <connection>scm:git:git@github.com:Azure/azure-sdk-for-java.git</connection>
-    <tag>HEAD</tag>
-  </scm>
+	<scm>
+		<url>scm:git:https://github.com/Azure/azure-sdk-for-java</url>
+		<connection>scm:git:git@github.com:Azure/azure-sdk-for-java.git</connection>
+		<tag>HEAD</tag>
+	</scm>
 
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <legal><![CDATA[[INFO] Any downloads listed may be third party software.  Microsoft grants you no rights for third party software.]]></legal>
-  </properties>
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<legal><![CDATA[[INFO] Any downloads listed may be third party software.  Microsoft grants you no rights for third party software.]]></legal>
+	</properties>
 
-  <developers>
-    <developer>
-      <id>microsoft</id>
-      <name>Microsoft</name>
-    </developer>
-  </developers>
+	<developers>
+		<developer>
+			<id>microsoft</id>
+			<name>Microsoft</name>
+		</developer>
+	</developers>
 
+	<dependencies>
+		<dependency>
+			<groupId>com.fasterxml.jackson.core</groupId>
+			<artifactId>jackson-databind</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>commons-codec</groupId>
+			<artifactId>commons-codec</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>com.google.guava</groupId>
+			<artifactId>guava</artifactId>
+			</dependency>
+		<dependency>
+			<groupId>com.microsoft.rest</groupId>
+			<artifactId>client-runtime</artifactId>
+			<version>1.3.0</version>
+		</dependency>
+		<dependency>
+			<groupId>com.microsoft.azure</groupId>
+			<artifactId>azure-client-authentication</artifactId>
+			<version>1.3.0</version>
+		</dependency>
+		<dependency>
+			<groupId>com.microsoft.azure</groupId>
+			<artifactId>azure-keyvault</artifactId>
+			<version>1.1-beta-1</version>
```

Comment:
Same comment as before here.  This is a lot of dependencies for JsonWebKeyCurveName.  Also, if you add it here you could reuse it since I believe azure-keyvault-cryptography already has a dependency on webkey